liquibase-core/src/main/java/liquibase/command/core/RegisterChangeLogCommand.java

```diff
@@ -107,6 +110,16 @@ protected CommandResult run() throws Exception {
             if (project == null) {
                 return new CommandResult("Project Id '" + hubProjectId + "' does not exist or you do not have access to it", false);
             }
+        } else if (hubProjectName != null) {
+            if (hubProjectName.length() > 255) {
```

Comment:
We probably shouldn't have validation logic in here that hub should be taking care of. Maybe need to improve the messaging we get from hub first, though?